mlir/lib/Transforms/Utils/DialectConversion.cpp

```diff
@@ -1520,7 +1520,7 @@ void ConversionPatternRewriterImpl::notifyOperationInserted(
 }
 
 void ConversionPatternRewriterImpl::notifyOpReplaced(
```

Comment:
This function is also a sink function i.e. needs to at the very least take `SmallVector<SmallVector<Value>>&&` such that we can move `repl` below into `map`